modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json

```diff
@@ -925,30 +926,167 @@
       "filename_template" : "https://ftp.ebi.ac.uk/pub/databases/eva/rs_releases/release_4/by_species/meleagris_gallopavo/Turkey_5.1/GCA_000146605.4_current_ids.vcf.gz"
     },
     {
-      "id": "collared_flycatcher_EVA",
-      "source_name": "EVA",
-      "description": "Variants from EVA",
-      "species": "ficedula_albicollis",
-      "assembly": "FicAlb1.5",
-      "type": "remote",
-      "use_as_source" : 1,
-      "filename_template": "https://ftp.ebi.ac.uk/pub/databases/eva/rs_releases/release_3/by_species/ficedula_albicollis/GCA_000247815.2/GCA_000247815.2_current_ids.vcf.gz",
-      "chromosomes": [
-          "1", "1A", "2", "3", "4", "4A", "5", "6", "7", "8", "9", "10",
-          "11", "12", "13", "14", "15", "17", "18", "19", "20", "21", "22",
-          "23", "24", "25", "26", "27", "28", "Z", "LG34", "LG35", "LGE22"
-      ]
-    },
-    {
-      "id": "sander_lucioperca_EVA",
-      "source_name": "EVA",
-      "description": "Variants from EVA",
-      "species": "sander_lucioperca",
-      "assembly": "SLUC_FBN_1",
-      "type": "remote",
-      "use_as_source" : 1,
-      "use_seq_region_synonyms": 1,
-      "filename_template": "https://ftp.ebi.ac.uk/pub/databases/eva/rs_releases/release_3/by_species/sander_lucioperca/GCA_008315115.1/GCA_008315115.1_current_ids.vcf.gz"
+        "id": "eva_ficedula_albicollis_gca000247815v2",
+        "description": "Variants from EVA",
+        "species": "Ficedula_albicollis",
+        "assembly": "FicAlb1.5",
+        "type": "remote",
+        "filename_template": "https://ftp.ebi.ac.uk/pub/databases/eva/rs_releases/release_4/by_species/ficedula_albicollis/GCA_000247815.2/GCA_000247815.2_current_ids.vcf.gz",
+        "use_as_source": 1,
+        "use_seq_region_synonyms": 0,
+        "strict_name_match": 0,
+        "source_name": "EVA",
+        "source_version": 4
+    },
+    {
+        "id": "eva_sander_lucioperca_gca008315115v1",
+        "description": "Variants from EVA",
+        "species": "Sander_lucioperca",
+        "assembly": "SLUC_FBN_1",
+        "type": "remote",
+        "filename_template": "https://ftp.ebi.ac.uk/pub/databases/eva/rs_releases/release_4/by_species/sander_lucioperca/GCA_008315115.1/GCA_008315115.1_current_ids.vcf.gz",
+        "use_as_source": 1,
+        "use_seq_region_synonyms": 0,
```

Comment:
For release 3, `use_seq_region_synonyms: 1`. No need to be `1` anymore?

Reply:
nope! they updated the vcf file.